GoogleForCreators/web-stories-wp
# Dashboard: Updated Entertainment template to 9:16

## Summary

This PR applies the 9:16 ratio updates to the Entertainment template.

## To-do

The ratio change in design means the preview story cards will have to be updated from 2:3 to 9:16 which will happen in another PR.

## User-facing changes

The Entertainment templates has multiple tweaks and layout updates.

## Testing Instructions

1.) Go to "Explore Templates"
2.) Click on the "Entertainment" template.
3.) Click on each page in the detail view and make sure they look good like the updated designs found in this [Figma doc](https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61).

Fixes #2474 

## Screenshots

<img src="https://user-images.githubusercontent.com/40646372/84714142-a5489380-af21-11ea-9be4-746669b56cea.png" height=200> <img src="https://user-images.githubusercontent.com/40646372/84715270-d24a7580-af24-11ea-92cc-71e9e2a4ede3.png" height=200> <img src="https://user-images.githubusercontent.com/40646372/84715285-db3b4700-af24-11ea-9771-7d6e0cb42a21.png" height=200> 

<img src="https://user-images.githubusercontent.com/40646372/84715301-e1c9be80-af24-11ea-824b-4922ce10ec19.png" height=200> <img src="https://user-images.githubusercontent.com/40646372/84715316-ea21f980-af24-11ea-85e1-da3ef2665fdf.png" height=200> <img src="https://user-images.githubusercontent.com/40646372/84715397-31a88580-af25-11ea-8659-0abc3505df19.png" height=200>

<img src="https://user-images.githubusercontent.com/40646372/84715409-39682a00-af25-11ea-8413-9211bda35918.png" height=200> <img src="https://user-images.githubusercontent.com/40646372/84715424-4258fb80-af25-11ea-8f6c-f77992e4c725.png" height=200> <img src="https://user-images.githubusercontent.com/40646372/84715441-4a18a000-af25-11ea-8449-0dbf4ec76d7b.png" height=200>
